lib/parsers/asm-parser-ewavr.ts

```diff
@@ -138,13 +160,15 @@ export class AsmEWAVRParser extends AsmParser {
             if (matches) {
                 currentLabel = {
                     lines: [],
-                    initialLine: currentLine,
+                    initialLine: currentLine || 0,
                     name: matches[1],
                     file: currentFile,
+                    require: [],
                 };
-                definedLabels[matches[1]] = currentLine;
+                definedLabels[matches[1]] = currentLine || 0;
                 resultObject.labels.push(currentLabel);
             }
+            return currentLabel;
```

Comment:
Required for a type hack?

Reply:
this was required to assign in the loop to make the loop not think that currentLabel didn't change during the loop.